pyanaconda/core/configuration/storage.py

```diff
@@ -142,3 +123,78 @@ def luks_version(self):
             raise ValueError("Invalid value: {}".format(value))
 
         return value
+
+    @property
+    def default_partitioning(self):
+        """Default partitioning.
+
+        Returns a list of dictionaries with mount point attributes.
+        The name of the mount point is represented by the attribute
+        'name' in the dictionary representation.
+
+        Valid attributes:
+
+            name       The name of the mount point.
+            size       The size of the mount point.
+            min        The size will grow from min size to max size.
+            max        The max size is unlimited by default.
+            free       The required available space.
+            encrypted  The mount point will be encrypted.
+
+        :return: a list of dictionaries with mount point attributes
+        """
+        return self._get_option("default_partitioning", self._convert_partitioning)
+
+    def _convert_partitioning(self, value):
+        """Convert a partitioning string into a list of dictionaries."""
+        return list(map(self._convert_partitioning_line, value.strip().split("\n")))
+
+    @classmethod
+    def _convert_partitioning_line(cls, line):
+        """Convert a partitioning line into a dictionary."""
+        # Parse the line.
+        name, attributes = cls._split_string(line)
+
+        attributes = attributes.strip("()").split(",")
```

Comment:
Please rename `attributes` and `attrs` to `raw_attrs` and `attrs` or the long variants. Point is to avoid having two attributes names here. Or another option is to name `attrs` as `result_attrs`.

Reply:
Fixed.